docs/developers/build-modular.md

```diff
@@ -4,7 +4,7 @@ sidebar_label: Build Modular
 
 # Build Modular
 
-This section will explain the advantages of building on
+This sections will explain the advantages of building on
```

Comment:
This setion or These sections

Reply:
a547078652d654f61aba1becdbce0d4bacc09b21